README.md

```diff
@@ -1 +1,110 @@
+- [ ] When we want to add one or more new ligands to the set (e.g. by adding rows to the [COVID Moonshot Compound Tracker](https://covid.postera.ai/covid/submissions/compounds), which can be [downloaded via the "Export CSV" button](https://covid.postera.ai/covid/submissions.csv) or at the associated [master GitHub repo](https://github.com/postera-ai/COVID_moonshot_submissions) in a [CSV file](https://github.com/postera-ai/COVID_moonshot_submissions/blob/master/covid_submissions_all_info.csv)), we convert the SMILES for the ligand into actual 3D molecules, expanding stereochemistry and store in SDF, and then identify which reference protein PDB structures we will create alchemical transformations (one transformation per FAH `RUN` for).  [JDC will provide script for this part, still being worked out.]
+Hannah currently uses a [script](https://github.com/choderalab/PersesAtFoldingAtHome/blob/master/SARS-CoV-2-Mpro/create-json.py) to populate a [JSON file](https://github.com/choderalab/PersesAtFoldingAtHome/blob/master/SARS-CoV-2-Mpro/aminopyridines%202020-07-04.json), but we don't need to use this format (unless Hannah says otherwise).
+- [ ] Hannah uses a script to parallelize the setup of individual transformations in parallel using our cluster (`lilac`) via an [LSF bash script](https://github.com/choderalab/PersesAtFoldingAtHome/blob/master/SARS-CoV-2-Mpro/submit-ligpairs.sh) and a [python `run.py`](https://github.com/choderalab/PersesAtFoldingAtHome/blob/master/SARS-CoV-2-Mpro/run.py).
+- [ ] To set up a single ligand transformation (`i -> j`), we run the `perses` command-line entry point `perses-fah config.yaml`, where `config.yaml` looks like this:
+```yaml
+protein_pdb: CDK2_protein.pdb
+ligand_file: CDK2_ligands_shifted.sdf
+old_ligand_index: 0
+new_ligand_index: 1
+forcefield_files:
+    - amber/ff14SB.xml # ff14SB protein force field
+    - amber/tip3p_standard.xml # TIP3P and recommended monovalent ion parameters
+    - amber/tip3p_HFE_multivalent.xml # for divalent ions
+    - amber/phosaa10.xml # HANDLES THE TPO
+atom_expression:
+    - IntType
+bond_expression:
+    - DefaultBonds
+complex_projid: 13418
+solvent_projid: 13419
+```
+In practice, this is generated from a JSON file with a script called `run.py`.
+This requires a GPU to be practical (e.g. GTX 1080).
+
+This will create files `{complex_projid}/RUNS/RUN/...` and `{solvent_projid}/RUNS/RUN/...` with the directories populated with some metadata important to retain plus `{system,state,integrator,core}.xml` (or `.bz2` or `.gz` compressed versions thereof).
+- [ ] `rsync` these files over to `server@aws3.foldingathome.org:/projects/available/covid-moonshot/`. It's OK to add more RUNs dynamically, but they must be compact and sequentially numbered from `RUN0`. The `{projid}` directory will be symlinked to `/home/server/server2/projects/{projid}` for the server to discover it.
```

Comment:
Should this be
```suggestion
- [ ] `rsync` these files over to `server@aws3.foldingathome.org:server2/projects/available/covid-moonshot/`. It's OK to add more RUNs dynamically, but they must be compact and sequentially numbered from `RUN0`. The `{projid}` directory will be symlinked to `/home/server/server2/projects/{projid}` for the server to discover it.
```

Reply:
It seems like it would be important to check that RUN number doesn't collide with one already on the server to avoid overwriting data, is that correct? In practice, should we look up the first unused RUN number when setting `trajectory_directory` in the yaml?

Reply:
Yes it would - so far we have avoided having clashing runs by using the `.json`s to keep an index